src/packages/tests/src/__tests__/__helpers__/integrationTest.ts

```diff
@@ -293,20 +298,13 @@ async function setupScenario(kind: string, input: Input, scenario: Scenario) {
 
   await ctx.fs.dirAsync('.')
 
-  if (input.database.name === 'sqlserver') {
-    state.db = await input.database.connect({ ...ctx, step: 'database' })
-    const databaseUpSQL = input.database.up?.(ctx) ?? ''
-    await input.database.send(state.db, databaseUpSQL)
-    await input.database.close?.(state.db)
-
-    const scenarioUpSQL = scenario.up
-    state.db = await input.database.connect({ ...ctx, step: 'scenario' })
-    await input.database.send(state.db, scenarioUpSQL)
+  state.db = await input.database.connect(ctx)
+  const databaseUpSQL = input.database.up?.(ctx) ?? ''
+  if (input.database.create) {
```

Comment:
Ideally adding a create method on the implementation wouldn't change the behaviour of the send function, but I couldn't think of another way to decide whether to send the up SQL and scenario SQL separately (Apart from checking `name === "sqlserver"` like we were doing before).

Reply:
I'm still of the opinion that we could:

- Call `create` if it exists
- Call send with the `ctx` argument, even if other DBs don't use (need) it.

That should retain behaviour of the `send` function (you'd still need to change the other `__databases` files though to add an unused param I think), and also make it so its signature remains the same across DBs.

Not a strong opinion though, this is kinda fine to me too.

Reply:
Just made this change, ctx is now a required param in send method. By passing in the db up sql and scenario sql separately to the send function as well, and then combining them in the implementation before making the query, we don't need a new create method at all :)